### ISSUE TITLE
SkinShuffle

### ISSUE THREAD

**Issue by IMB11** [skywardmc/additive#54]
**Describe the mod you'd like to add, remove, or change**

https://modrinth.com/mod/skinshuffle
Features:

- Auto Reconnect if the server doesn't have a SkinShuffle compatible plugin or SkinShuffle itself installed.
- Creating skin presets from usernames, UUIDs, URLs, files and built-in skins via resource packs.
- Easy to use interface with clear instructions for non-hotreloadable servers.
- Hot Reload if SkinShuffle or SkinShuffle-compatible plugin is installed on server.
- Hot Reload on LAN (if you're host, or if host has skinshuffle) + Singleplayer worlds.
- Carousel and Compact Grid Layout modes (for smaller and larger screens, or larger and smaller collections of skins!)
- High configurability, can hide skin change widget from title screen/pause screen (can still get to preset change screen via mod menu), can modify the way skins are rendered (follow cursor or rotate on the spot!)

**TLDR:** Change skin in-game without the need to relog manually, or tab out of Minecraft and go to the Minecraft.net/account/skin page. Allows quick switching, great for content creators who are making videos.

**List alternative mods or solutions you've considered**

There isn't really any alternatives to SkinShuffle that are so broad, it's pretty unique.

**Additional context**

![image-23](https://github.com/skywardmc/additive/assets/93472213/57f3fcc4-3453-4d17-a4b9-ba1ee6c72770)
![image-31](https://github.com/skywardmc/additive/assets/93472213/d07727f6-156d-4679-94c2-b1b2d9489095)
![image-25](https://github.com/skywardmc/additive/assets/93472213/389dd4d6-fa9d-4006-b302-14274ee5bd88)
![e13b4fcd7b37d02aa043e422d2e9560d6f099527](https://github.com/skywardmc/additive/assets/93472213/d5e53d26-e842-4dc9-b58a-84bd440af4a5)
![d1f435ad282114885fd9b589c538c8e7250b39ac](https://github.com/skywardmc/additive/assets/93472213/2ed7efbb-dbb7-4625-b49b-16fe1ef37905)
![0b5e81688937e2685b2fa00ec709c5c1db302648](https://github.com/skywardmc/additive/assets/93472213/3a992e9e-a70c-49d3-8ea1-27c6015a4d2f)
![bef30a864813602bf89d08fe6d103384c867a219](https://github.com/skywardmc/additive/assets/93472213/e9449645-ea17-40aa-b4a9-fb719cd6460a)

**Comment by intergrav** [comment]
Personally I feel that this is a bit out of scope for Additive since it is primarily meant to be an OptiFine alternative - I'll let other people voice their opinions here though

**Comment by intergrav** [comment]
> I'll let other people voice their opinions here though

Marking issue as not planned as there was no activity for a while. You can still comment here if you want though
